samples/bluetooth/mesh/prj.conf

```diff
@@ -21,6 +20,7 @@ CONFIG_BT_PERIPHERAL=y
 CONFIG_BT=y
 CONFIG_BT_TINYCRYPT_ECC=y
 CONFIG_BT_L2CAP_TX_BUF_COUNT=5
+CONFIG_BT_MAX_CONN=2
```

Comment:
Why was this changed?

Reply:
Default this is set to 1, and I found out this need to be 2. It is 1 when CONFIG_BT_OBSERVER=y and CONFIG_BT_PERIPHERAL=y, but 2 if CONFIG_BT_CENTRL=y and CONFIG_BT_PERIPHERAL=y is set.

Reply:
Why would someone set CONFIG_BT_CENTRAL in a mesh sample? It does not look like any functionality in this sample needs BT_CENTRAL.

Reply:
due to gatt proxy client?

Reply:
Yeah, but Central role is not enabled here, neither PB GATT or Proxy Client. Unless I miss something...

Reply:
PB_GATT is added 5 lines down.

Reply:
But not client

Reply:
Just check if you really need BT_MAX_CONN=2, by flashing and provisioning the sample. The CENTRAL should not be selected, if it is getting selected there is some misconfiguration.

Reply:
This was not necessary to be 2.